### DISCUSSION TITLE
Find and replace?

### DISCUSSION THREAD

**Discussion by t6489609** [zed-industries/zed#6473] [Feature Requests]
I was trying to find the find/replace/replace-all command.
Does it exist?
If not, I guess there is another way to do this. What is it?

Thanks.

Jason Koh

**Comment by JosephTLyons** [comment]
We don't currently have replace functionality; it's on our roadmap to add it.  You can sort of emulate find and replace within a single file by using the `editor: select next` command, which will highlight all occurrences of whatever text you have highlighting, then you can replace all by simply typing.  This doesn't get you everything you need for find and replace, but can be helpful in many cases.

**Comment by notpeter** [comment]
We support this now.  Replace all is the R with the * next to it:
<img width="178" alt="image" src="https://github.com/zed-industries/zed/assets/145113/80037382-09ff-45cb-b32a-cab74eef43ba">
